staff/sys/makevm

```diff
@@ -242,6 +257,27 @@ def _main(args):
             print('Cancelled.')
             sys.exit(2)
 
+    if args.ceph and not shutil.which('ceph'):
+        print("Warning: You are trying to use Ceph but I couldn't find the `ceph` executable.")
+
+        if not confirm():
+            print('Cancelled.')
+            sys.exit(2)
+
+    if args.ceph and args.vg != 'vg':
```

Comment:
we should store the default value in a global variable or something so we only have to change it in one place

Reply:
use `parser.get_default`